src/App.js

```diff
@@ -1,18 +1,22 @@
+import React from "react";
 import TodoList from "./TodoList";
 import AddTodoForm from "./AddTodoForm";
 
 function App() {
-  const NewTodo = (props) => {
-    const [newTodo,setNewTodo] = React.useState('');
-  }
+ 
+
+    const[todoList, setTodoList]= React.useState([]);
+    const addTodo =(newTodo)=>{
```

Comment:
Be careful of spacing. Here are a couple of areas that have spacing errors, make sure you know what the problems are: 
- `const[todoList`
- `=(newTodo)=>{`